src/clj/puppetlabs/puppetserver/jruby_request.clj

```diff
@@ -84,11 +64,11 @@
     (let [environment (get-environment-from-request request)]
       (cond
         (nil? environment)
-        (throw-bad-request!
+        (mw/throw-bad-request!
```

Comment:
I think the docstring needs to be updated  - `:bad-request slingshot exception is thrown` rather than `::bad-request slingshot exception is thrown`.